library/Icinga/Crypt/AesCrypt.php

```diff
@@ -207,21 +261,21 @@ public function decryptFromBase64($data)
     }
 
     /**
-     * Encrypt the given data using the key, iv and tag
+     * Encrypt the given string
      *
      * @param string $data
      *
-     * @return string encrypted data
+     * @return string encrypted string
      *
      * @throws RuntimeException If decryption fails
      */
     public function encrypt($data)
     {
-        if ($this->method === 'AES-128-CBC') {
-            return $this->encryptCBC($data);
+        if (! $this->isAuthenticatedEncryptionRequired()) {
+            return $this->nonAEEncrypt($data);
         }
 
-        $encrypt = openssl_encrypt($data, $this->method, $this->getkey(), 0, $this->getIV(), $this->tag);
+        $encrypt = openssl_encrypt($data, $this->getMethod(), $this->getkey(), 0, $this->getIV(), $this->tag);
```

Comment:
Please use the correct casing for `getKey`

Reply:
How is that even possible 🤔 . I always use `PhpStorm's` autocomplete.